<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>w_sys_manage_basicdataimport_m_Dialog</class>
 <widget class="QDialog" name="w_sys_manage_basicdataimport_m_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>471</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>280</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>跳过第一行</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>115</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>会员号</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>115</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>姓名</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>115</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>余额</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>115</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>积分</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>115</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>有效期</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_no">
   <property name="geometry">
    <rect>
     <x>141</x>
     <y>40</y>
     <width>50</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_name">
   <property name="geometry">
    <rect>
     <x>141</x>
     <y>70</y>
     <width>50</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_num">
   <property name="geometry">
    <rect>
     <x>141</x>
     <y>100</y>
     <width>50</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_point">
   <property name="geometry">
    <rect>
     <x>141</x>
     <y>130</y>
     <width>50</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_date">
   <property name="geometry">
    <rect>
     <x>141</x>
     <y>160</y>
     <width>50</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>7</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>115</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>分隔符</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>190</y>
     <width>50</width>
     <height>27</height>
    </rect>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
   <item>
    <property name="text">
     <string>,</string>
    </property>
   </item>
  </widget>
  <widget class="QCheckBox" name="checkBox_2">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>280</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>导入有效期</string>
   </property>
  </widget>
  <widget class="lds_datetimeEdit" name="dateTimeEdit">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>160</y>
     <width>181</width>
     <height>27</height>
    </rect>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_import">
   <property name="geometry">
    <rect>
     <x>378</x>
     <y>390</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>开始导入</string>
   </property>
   <property name="outer_stylesheet" stdset="0">
    <string>pushbutton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>390</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
   <property name="outer_stylesheet" stdset="0">
    <string>pushbutton</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>100</y>
     <width>50</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>列</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>130</y>
     <width>50</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>列</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>70</y>
     <width>50</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>列</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>160</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>列</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>40</y>
     <width>50</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>列</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_path">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>10</y>
     <width>441</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>10</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>文件位置</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>10</y>
     <width>41</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="outer_stylesheet" stdset="0">
    <string>pushbutton</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>330</y>
     <width>601</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#ff546b;</string>
   </property>
   <property name="text">
    <string>列数最小是:0(-1表示跳过); 导入表中，每列数据不能包含分隔符</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>40</y>
     <width>115</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>会员类型</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_ch_typeno">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>40</y>
     <width>131</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>默认类别</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>70</y>
     <width>50</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>列</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_birthday">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>70</y>
     <width>50</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>-1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>70</y>
     <width>115</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>生日</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>100</y>
     <width>115</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>手机</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_handtel">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>100</y>
     <width>50</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>-1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>100</y>
     <width>50</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>列</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_19">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>130</y>
     <width>50</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>列</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_address">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>130</y>
     <width>50</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>-1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_20">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>130</y>
     <width>115</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>地址</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_ch_cardflag">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>190</y>
     <width>131</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>已发卡</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_21">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>190</y>
     <width>115</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>持卡状态</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_3">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>280</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>导入会员类型编号</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_22">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>40</y>
     <width>50</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>列</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_ch_typeno2">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>40</y>
     <width>50</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>-1</string>
   </property>
  </widget>
  <widget class="lds_ComboBox" name="comboBox_codec">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>390</y>
     <width>85</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>UTF-8</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>GBK</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_23">
   <property name="geometry">
    <rect>
     <x>129</x>
     <y>390</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>编码</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>lds_datetimeEdit</class>
   <extends>QDateTimeEdit</extends>
   <header>lds_datetimeedit.h</header>
  </customwidget>
  <customwidget>
   <class>lds_ComboBox</class>
   <extends>QComboBox</extends>
   <header>lds_combobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>w_sys_manage_basicdataimport_m_Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>381</x>
     <y>318</y>
    </hint>
    <hint type="destinationlabel">
     <x>481</x>
     <y>266</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
